### ISSUE TITLE
Breaks LSP documentation

### ISSUE THREAD

**Issue by kevinhughes27** [sunjon/Shade.nvim#25]
It seems this plugin breaks the LSP documentation from showing:
![image](https://user-images.githubusercontent.com/1965489/141698732-419fad28-bd0e-41d6-80b1-fd66466bea1d.png)

It took me a while to track down what the cause was - by systematically removing bits of my config until I removed this plugin and bam the documentation was working again. It could be a combination with the rest of my setup as well. Happy to help debug further just let me know what you need.

I like the idea of this plugin! It makes it nice to see which split I am/was working in :)

**Comment by nyngwang** [comment]
This is a troublesome plugin and I recommend not using it. It seems like the author has no time to maintain it.

**Comment by BStephenBB** [comment]
> This is a troublesome plugin and I recommend not using it. It seems like the author has no time to maintain it.

Bummer.

@nyngwang Is there a good alternative that is actively maintained that you'd recommend?

**Comment by nyngwang** [comment]
@BStephenBB No. I used to like the effect very much. But I uninstalled it after some troubles it caused when using with a more important plugin `nvim-tree.lua`.

**Comment by sunjon** [comment]
Hi, sorry for the delay in getting to this.
I don't use cmp - can you show me what the expected result is? It's hard to tell what the problem is from that single screenshot.

**Comment by kevinhughes27** [comment]
The UI element the arrow is pointing to doesn't show up when this plugin is used. The first part of the auto complete still shows but no the second part that includes more details about the selection.
